examples/sample_get_identity_buckets.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+import sys
+
+from uid2_client import IdentityMapClient
+
+
+# this sample client takes date time as input and generates an IdentityBucketsResponse object which contains
+# a list of buckets
+
+def _usage():
+    print('Usage: python3 sample_get_identity_buckets.py <base_url> <api_key> <client_secret> <year> <month> <day> <hour> <minute> <second>'
+          , file=sys.stderr)
+    sys.exit(1)
+
+
+if len(sys.argv) <= 9:
+    _usage()
+
+base_url = sys.argv[1]
+api_key = sys.argv[2]
+client_secret = sys.argv[3]
+year = int(sys.argv[4])
+month = int(sys.argv[5])
+day = int(sys.argv[6])
+hour = int(sys.argv[7])
+minute = int(sys.argv[8])
+second = int(sys.argv[9])
+
+client = IdentityMapClient(base_url, api_key, client_secret)
+
+identity_buckets_response = client.get_identity_buckets(datetime.datetime(year, month, day, hour, minute, second))
+
+if identity_buckets_response.buckets:
+    bucket = identity_buckets_response.buckets[0]
+    print("The bucket id of the first bucket: ", bucket.get_bucket_id())
+    print("The last updated timestamp of the first bucket: ", bucket.get_last_updated())
+else:
+    print("No buckets were returned for this datetime")
```

Comment:
i presume if the returned http status isn't 200 but errors, raise_for_status already raised an exception and abort this already so if this line is reached, http request was successful and there is really no bucket returned?

Reply:
When the list identity_buckets_response.buckets is empty, accessing identity_buckets_response.buckets[0] will throw an exception. Therefore, I created an if-else condition to handle situations where no buckets are returned.

This can occur even if the HTTP status is 200, such as when the input date is one day ahead of today. In this case, the response is still successful but contains an empty buckets list, which would return this line.

 {"body":[],"status":"success"}